### PR TITLE
Infra 156

### PR DESCRIPTION
**Pull request by josacar**
This change is [Reviewable](https://reviewable.io/reviews/peertransfer/glare/1)

### CODE REVIEW

**Comment by pollosp** [comment]
👍 nice refactor.

**Comment by pollosp** [comment]
👍 Nice refactor here.

File: lib/glare/cf_dns_records.rb [hunk 1]
```diff
@@ -25,32 +40,23 @@ def count
...
     def each
       @records.each { |record| yield(record) }
     end
 
-    def records_to_delete(targer_number)
-      records_to_delete = count - targer_number
-      return [] if records_to_delete < 0
+    def records_to_delete(target_number)
+      records_to_delete = count - target_number
+      return CfDnsRecords.new([]) if records_to_delete < 0
```
Comment (alvarobp) [inline lib/glare/cf_dns_records.rb]: Have you considered creating this empty collection in a factory method too? E.g. ``CfDnsRecords.empty``

**Comment by alvarobp** [comment]
👍

File: lib/glare/cf_dns_records.rb (hunk 1 above)
Comment (josacar) [inline lib/glare/cf_dns_records.rb]: I'll do, thanks!

**Pull request merged**